recipes-apps/factest/files/mz/buttons_and_leds_test.sh

```diff
@@ -0,0 +1,246 @@
+#!/bin/sh -e
+
+DEBUG=0
+
+source /usr/local/bin/gpio/gpio_common.sh
+
+cd /sys/class/gpio
+
+error_count=0
+
+
+echo "*** TEST MIO_SWS and MIO_LEDS"
+echo "***"
+
+# MIO LEDs (PCB D9, D8, D7, D6)
+MIO_LED_1_NB=`get_gpio MIO_LED_1`
+MIO_LED_2_NB=`get_gpio MIO_LED_2`
+MIO_LED_3_NB=`get_gpio MIO_LED_3`
+MIO_LED_4_NB=`get_gpio MIO_LED_4`
+
+# PS DIP switches (PCB SW4)
+MIO_SW_1_NB=`get_gpio MIO_SW_1`
+MIO_SW_2_NB=`get_gpio MIO_SW_2`
+MIO_SW_3_NB=`get_gpio MIO_SW_3`
+MIO_SW_4_NB=`get_gpio MIO_SW_4`
+
+process_mio_sw_to_leds() {
```

Comment:
I presume there isn't need for a EMC unattended style test as the MicroZed (Zynq-7000) is already been approved?